### DISCUSSION TITLE
Support for Vuggestue

### DISCUSSION THREAD

**Discussion by MikkelFrid** [scaarup/aula#185] [Ideas]
Hi 

I was wondering if it would be possible to add support for Vuggestue? 
More specifically the data on the 'Komme/Gå' tab, including info such as:
- Status (Kommet, Gået)
- Kom klokken <time>
- Gået klokken <time>
- Sovetider <time start, time end>
- Hentes <Name>

Look forward to hear, what is possible :)

**Comment by scaarup** [comment]
Hej @MikkelFrid 
Hvis vuggestuen bruger Aula og "Komme/gå" modulet, burde denne info da allerede være der?

**Comment by MikkelFrid** [comment]
Hej @scaarup 
Det var blot mig der var utålmodig, alt info kom efter lidt tid på den entity. 
Så jo, alt virker. Tak

**Reply by scaarup** [reply]
Super 👍
